packages/grid/_modules_/grid/constants/eventsConstants.ts

```diff
@@ -56,11 +56,11 @@ export const GRID_ROWS_SCROLL_END = 'scroll:rowEnd';
 export const GRID_COL_RESIZE_START = 'colResizing:start';
 export const GRID_COL_RESIZE_STOP = 'colResizing:stop';
 
-export const GRID_COL_REORDER_START = 'colReordering:dragStart';
-export const GRID_COL_REORDER_DRAG_OVER_HEADER = 'colReordering:dragOverHeader';
-export const GRID_COL_REORDER_DRAG_OVER = 'colReordering:dragOver';
-export const GRID_COL_REORDER_DRAG_ENTER = 'colReordering:dragEnter';
-export const GRID_COL_REORDER_STOP = 'colReordering:dragStop';
+export const GRID_COLUMN_REORDER_START = 'colReordering:dragStart';
```

Comment:
I renamed them because I understood that the convention here was to avoid abbreviated words. We already have `GRID_COLUMN_HEADER_CLICK`, so it would be natural to use the same prefix. The events for column resizing could be renamed in the `useGridColumnResize` refactoring.